packages/StatusTracker/src/components/Point/Point.styles.js

```diff
@@ -0,0 +1,108 @@
+import styled, { css } from "styled-components";
+import Button from "@paprika/button";
+import PaprikaPopover from "@paprika/popover";
+import tokens from "@paprika/tokens";
+import { fontSize, spacer } from "@paprika/stylers/lib/helpers";
+import * as types from "../../types";
+
+const passedPath = css`
+  border-bottom: 2px solid ${tokens.color.greenLighten10};
+`;
+
+const futurePath = css`
+  border-bottom: 2px dashed ${tokens.color.blackLighten20};
+`;
+
+export const PointWrapper = styled.li`
+  align-items: center;
+  display: inline-flex;
+  flex-grow: 1;
+  list-style: none;
+
+  &::after {
+    content: "";
+    display: inline-block;
+    flex-grow: 1;
+    margin: 0;
+    vertical-align: middle;
+
+    ${({ kind }) => (kind === types.kinds.PAST ? passedPath : futurePath)}
+  }
+
+  &:last-child {
+    flex-grow: 0;
+
+    ::after {
+      content: none;
+    }
+  }
+`;
+
+const basicPoint = css`
```

Comment:
`nit` currentPoint?

Reply:
This has been shared in both past point and future point